linux_os/guide/system/software/integrity/crypto/configure_crypto_policy/oval/shared.xml

```diff
@@ -1,7 +1,10 @@
 {{%- if target_oval_version == [5, 11] -%}}
 {{# there is no good alternative for symlink_test for OVAL 5.10 #}}
 {{%- macro crypto_policy_symlink_criterion(library) %}}
-      <criterion comment="Check if /etc/crypto-policies/back-ends/{{{ library }}}.config is linked to /usr/share/crypto-policies/POLICY/{{{ library }}}.txt" test_ref="test_crypto_policy_{{{ library }}}_symlink" />
+      <criteria operator="OR">
```

Comment:
What is the expected behavior if both criterion evaluate to true?

Presence of drop in `local.d/library.config` file and `back-end/library.config`being a symlink would indicates that `update-crypto-policies` was not run recently, and remediation should be run.
So appropriate operator for criteria would be `XOR`.

Reply:
The approach to explicitly check for them also works.